tests/devices/test_default_qubit_jax.py

```diff
@@ -475,6 +475,23 @@ def circuit(param):
         res = jacobian_transform(circuit)(param)
         assert jnp.allclose(res, jnp.zeros(wires ** 2))
 
+    def test_inverse_operation_jacobian_backprop(self, tol):
+        """Test that inverse operations work in backprop
+        mode"""
+        dev = qml.device('default.qubit.jax', wires=1)
+
+        @qml.qnode(dev, diff_method="backprop", interface="jax")
+        def circuit(param):
+            qml.RY(param, wires=0).inv()
+            return qml.expval(qml.PauliX(0))
+
+        x = 0.3
+        res = circuit(x)
+        assert np.allclose(res, -np.sin(x), atol=tol, rtol=0)
+
+        grad = jax.grad(lambda a: circuit(a).reshape(()))(x)

```

Comment:
@Thenerdstation this confused me for a while, until I realised that we currently still run this test with the old core _and_ the new core.

The old core didn't squeeze QNode output, but the new core does, so we'll be able to remove this wrapping soon :+1:

Reply:
👍